ring_buffer/ring_buffer.py

```diff
@@ -5,7 +5,15 @@ def __init__(self, capacity):
     self.storage = [None]*capacity
 
   def append(self, item):
+    self.storage[self.current % self.capacity] = item
+    self.current += 1
     pass
 
   def get(self):
```

Comment:
In the requirements for MVP there was an instruction asking to ta ke care of the cases where your storage has any None value, an those shouldn't be sent or returned, while your solution passes the tests, its not accountable for this case.